src/Mindee/Http/MindeeApi.cs

```diff
@@ -159,13 +159,6 @@ private static void AddPredictRequestParameters(PredictParameter predictParamete
             }
         }
 
-        private static string BuildUserAgent()
```

Comment:
Don't delete this, it makes the code harder to read. Just call the method as needed.